xbmc/guilib/GUITextLayout.h

```diff
@@ -185,9 +185,9 @@ class CGUITextLayout
   {
     return (letter & 0xffff) == L' ';
   };
-  inline bool CanWrapAtLetter(character_t letter) const XBMC_FORCE_INLINE
+  inline bool CanWrapAtLetter(const character_t& letter) const XBMC_FORCE_INLINE
```

Comment:
Passing by reference seems less efficient, character_t is uint32_t?

Reply:
removed, i thought reusing the same memory was more efficent